src/main/java/nl/tudelft/dnainator/ui/views/View.java

```diff
@@ -21,6 +21,9 @@
  */
 public class View extends Pane {
 	private static final double SCALE = .1;
+	private static final double ZOOM_FACTOR = 1e-3;
+	private static final int DEFAULT_ZOOM_IN = 55;
```

Comment:
Because having equally large steps makes the amount that is zoomed in differ from each other.